kedro-airflow/tests/conftest.py

```diff
@@ -15,7 +16,10 @@
 @fixture(name="cli_runner")
 def cli_runner():
     runner = CliRunner()
+    owd = Path.cwd()
     with runner.isolated_filesystem():
+        fp = owd / "kedro_airflow/airflow_dag_template.j2"
+        copyfile(fp.resolve(), Path("./airflow_dag.j2").resolve())
```

Comment:
What is this for? 🤔 

Edit: is this the thing you're trying to fix by altering the CircleCI config instead? If it is possible to do so that way then it would definitely be preferable I think.

Reply:
No this is different. My reasoning here was that I wanted to provide a "real" different filepath to test `--jinja-file`. Therefore I am copying the template file into the test directory under a different filepath. I could of course just test the `--jinja-file` option with the original template file in the kedro_airflow directory but I wanted to be a bit more thorough. Maybe its not necessary 🤷

Reply:
Ahh I see, thanks for the explanation. This makes sense and I think is a good thing to do 👍  My only very minor comment would be that `owd` seems like a bit of an odd variable name - why not `cwd`?